tracer/src/Datadog.Trace/Debugger/Expressions/ProbeProcessor.cs

```diff
@@ -69,6 +69,19 @@ private void InitializeProbeProcessor(ProbeDefinition probe)
 
             SetExpressions(probe);
 
+            var capture = (probe as LogProbe)?.Capture;
+            var maxInfo = capture != null
+                ? new MaxInfo(
+                    MaxReferenceDepth: capture.Value.MaxReferenceDepth <= 0 ? DebuggerSettings.DefaultMaxDepthToSerialize : capture.Value.MaxReferenceDepth,
+                    MaxCollectionSize: capture.Value.MaxCollectionSize <= 0 ? DebuggerSettings.DefaultMaxNumberOfItemsInCollectionToCopy : capture.Value.MaxCollectionSize,
+                    MaxFieldCount: capture.Value.MaxFieldCount <= 0 ? DebuggerSettings.DefaultMaxNumberOfFieldsToCopy : capture.Value.MaxFieldCount,
+                    MaxLength: capture.Value.MaxLength <= 0 ? DebuggerSettings.DefaultMaxStringLength : capture.Value.MaxLength)
+                : new MaxInfo(
```

Comment:
Can be null for other probes than `LogProbe` for clarity or better with a default value? AFAIK, we currently do not use these values in probes other than `LogProbe`.

Reply:
I use default values here:

https://github.com/DataDog/dd-trace-dotnet/pull/5364/files/2fc915b1159617f9e04f5224ea926c6f8f31c323#diff-055d92892a586a3741ac8cbf70a7aea123046f8ba55d6ae87acc6db621b06cf6:~:text=Resolve%20conversation-,MaxReferenceDepth%3A%20DebuggerSettings.DefaultMaxDepthToSerialize%2C,MaxLength%3A%20DebuggerSettings.DefaultMaxStringLength)%3B,-ProbeInfo%20%3D%20new